test/TestExtensions/GenerateLicenseList/LicenseDataParser.cs

```diff
@@ -91,6 +92,11 @@ private static bool ReadBool(JToken json, string property)
             return json[property].Value<bool>();
         }
 
+        private static bool ReadBool(JToken json, string property, bool defaultValue = false)
+        {
+            return json.Value<bool?>(property) ?? defaultValue;
```

Comment:
did you notice any problems parsing the file or?

Reply:
No, updated `NuGetLicenseData.cs` is included in this PR.

Reply:
The reason for adding that method was that the `isFsfLibre` flag is not present on every license entry in the source json file, so I had to add a `ReadBool` overload with default fallback.